pynxtools/dataconverter/convert.py

```diff
@@ -37,7 +37,15 @@
 if sys.version_info >= (3, 10):
     from importlib.metadata import entry_points
 else:
-    from importlib_metadata import entry_points
+    try:
+        from importlib_metadata import entry_points
+    except ImportError:
+        # If importlib_metadata is not present
+        # we provide a dummy function just returning an empty list.
+        # pylint: disable=W0613
+        def entry_points(group):
```

Comment:
Here I did some minor modification so that, nomad CI/CD pipeline pass the all test. No issue in nomad.